wdl2cwl/tests/oop_cwl_files/deepvariant.cwl

```diff
@@ -0,0 +1,136 @@
+class: CommandLineTool
+id: RunDeepVariant
+inputs:
+  - id: referenceFasta
+    type: File
+  - id: referenceFastaIndex
+    type: File
+  - id: inputBam
+    type: File
+  - id: inputBamIndex
+    type: File
+  - id: modelType
+    type: string
+  - id: outputVcf
+    type: string
+  - id: postprocessVariantsExtraArgs
+    type:
+      - string
+      - 'null'
+  - id: customizedModel
+    type:
+      - File
+      - 'null'
+  - id: numShards
+    type:
+      - int
+      - 'null'
+  - id: outputGVcf
+    type:
+      - string
+      - 'null'
+  - id: outputGVcfIndex
+    type:
+      - string
+      - 'null'
+  - id: regions
+    type: string
+  - id: sampleName
+    type:
+      - string
+      - 'null'
+  - id: VCFStatsReport
+    default: true
+    type:
+      - boolean
+      - 'null'
+  - id: memory
+    default: 3G
+    type: string
+  - id: timeMinutes
+    default: 5000
+    type: int
+  - id: dockerImage
+    default: google/deepvariant:1.0.0
+    type: string
+outputs:
+  - id: outputVCF
+    type: File
+    outputBinding:
+        glob: $(inputs.outputVcf)
+  - id: outputVCFIndex
+    type: File
+    outputBinding:
+        glob: $(inputs.outputVcf + ".tbi")
```

Comment:
```suggestion
        glob: $(inputs.outputVcf).tbi
```
As shown in the non OOP version: https://github.com/common-workflow-lab/wdl-cwl-translator/blob/729faea7fdef86b8dcd47997fe03320045d36473/wdl2cwl/tests/cwl_files/deepvariant.cwl#L64